great_expectations/data_context/store/ge_cloud_store_backend.py

```diff
@@ -29,7 +29,8 @@ class GeCloudRESTResource(str, Enum):
     EXPECTATION_VALIDATION_RESULT = "expectation_validation_result"
     PROFILER = "profiler"
     RENDERED_DATA_DOC = "rendered_data_doc"
-    SUITE_VALIDATION_RESULT = "suite_validation_result"
+    SUITE_VALIDATION_RESULT = "suite_validation_result"  # Deprecated
```

Comment:
Keeping the old resource around due to the need for legacy behavior with E2E tests - this should be stripped out but wanted to start a conversation around it before I did anything. Same throughout.